File: VB/ShapeTitleOptions/Form1.vb
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraMap

Namespace ShapeTitleOptions
    Partial Public Class Form1
        Inherits Form

        Private dot As MapDot

        Public Sub New()
            InitializeComponent()
        End Sub

        #Region "#TitleOptions"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load
            Dim storage As New MapItemStorage()
            Dim layer As New VectorItemsLayer() With {.Data = storage}

            dot = New MapDot() With {.Size = 100}
            dot.TitleOptions.TextColor = Color.Orange
            dot.TitleOptions.TextGlowColor = Color.Black
            dot.TitleOptions.Pattern = "{Title}"

            dot.Attributes.Add(New MapItemAttribute() With {.Name = "Title", .Value = "Hello," & ControlChars.Lf & "I am Dot."})
            storage.Items.Add(dot)

            mapControl1.Layers.Add(layer)
        End Sub
        #End Region ' #TitleOptions

        #Region "#TitleVisibility"
        Private Sub cbTitleVisibility_CheckedChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles cbTitleVisibility.CheckedChanged
            If cbTitleVisibility.Checked Then
                dot.TitleOptions.Visibility = VisibilityMode.Visible
            Else
                dot.TitleOptions.Visibility = VisibilityMode.Hidden
            End If
        End Sub
        #End Region ' #TitleVisibility
    End Class
End Namespace

File: CS/ShapeTitleOptions/Form1.cs
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraMap;

namespace ShapeTitleOptions {
    public partial class Form1 : Form {
        MapDot dot;
        
        public Form1() {
            InitializeComponent();
        }

        #region #TitleOptions
        private void Form1_Load(object sender, System.EventArgs e) {
            MapItemStorage storage = new MapItemStorage();
            VectorItemsLayer layer = new VectorItemsLayer() {
                Data = storage
            };

            dot = new MapDot() { Size = 100 };
            dot.TitleOptions.TextColor = Color.Orange;
            dot.TitleOptions.TextGlowColor = Color.Black;
            dot.TitleOptions.Pattern = "{Title}";

            dot.Attributes.Add(new MapItemAttribute() { Name = "Title", Value = "Hello,\nI am Dot." });
            storage.Items.Add(dot);

            mapControl1.Layers.Add(layer);
        }
        #endregion #TitleOptions

        #region #TitleVisibility
        private void cbTitleVisibility_CheckedChanged(object sender, System.EventArgs e) {
            if (cbTitleVisibility.Checked)
                dot.TitleOptions.Visibility = VisibilityMode.Visible;
            else
                dot.TitleOptions.Visibility = VisibilityMode.Hidden;
        }
        #endregion #TitleVisibility
    }
}
